Install instructions do not work on Ubuntu Trusty with Mirage installed via OPAM

Current instructions are:
- Install Mirage
- `cd slides && make`

Mirage seems to install just fine with OPAM. Result `cd slides & make` on Ubuntu Trusty gives this result:

```
$ make
ocaml setup.ml -configure --enable-unix --disable-xen
ocamlfind: Package `cohttp.mirage' not found
W: Field 'pkg_cohttp_mirage' is not set: Command ''~/.opam/system/bin/ocamlfind' query -format %d cohttp.mirage > '/tmp/oasis-f27aa2.txt'' terminated with error code 2
ocamlfind: Package `uri' not found
W: Field 'pkg_uri' is not set: Command ''~/.opam/system/bin/ocamlfind' query -format %d uri > '/tmp/oasis-ed44e7.txt'' terminated with error code 2
ocamlfind: Package `cow' not found
W: Field 'pkg_cow' is not set: Command ''~/.opam/system/bin/ocamlfind' query -format %d cow > '/tmp/oasis-46c2b2.txt'' terminated with error code 2
E: Cannot find findlib package cohttp.mirage
E: Cannot find findlib package cow (>= 0.3.2)
E: Cannot find findlib package uri
E: Failure("3 configuration errors")
make: *** [all] Error 1
```

`opam install cow uri` seems fine. `opam install cohttp.mirage` results in this error:

```
$ opam install cohttp.mirage
Creating a cache of metadata in ~/.opam/state.cache ...
[ERROR] The package "cohttp" has no version mirage.
'opam install cohttp.mirage' failed.
```

--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 IOCAML ?= iocaml -completion ./notebooks
 
 MODE  ?= unix
-NET   ?=
+NET   ?= socket
 MIRAGE = mirage
 
 .PHONY: all configure build run depend clean docs

--- a/README.md
+++ b/README.md
@@ -1,10 +1,12 @@
-This is the content for the Commercial Users of Functional Programming
-(CUFP 2011) tutorial on "Building a Functional OS".
+This is tutorial content for Mirage OS, written as a self-hosting
+slide deck and some IOCaml interactive browser notebooks.
 
 To view the content:
 
 * [Install Mirage](http://www.openmirage.org/wiki/install)
-* `cd slides && make`
+* `make configure`
+* `make build`
 
-And navigate to `http://127.0.0.1:8000` on either Chrome or Safari
-(other browsers may work, but your mileage may vary with the SVG).
+Set the `MODE` variable to `xen` or `unix` (default `unix`).
+
+And navigate to `http://127.0.0.1:8080` to view the slides.

--- a/src/config.ml
+++ b/src/config.ml
@@ -41,9 +41,9 @@ let stack console =
 
 let port =
   try match Sys.getenv "PORT" with
-    | "" -> 80
+    | "" -> 8080
     | s  -> int_of_string s
-  with Not_found -> 80
+  with Not_found -> 8080
 
 let server =
   http_server port (stack default_console)

